src/libraries/System.Net.Http/src/System/Net/Http/BrowserHttpHandler/BrowserHttpHandler.cs

```diff
@@ -95,14 +116,27 @@ public bool PreAuthenticate
 
         public bool AllowAutoRedirect
         {
-            get => throw new PlatformNotSupportedException();
-            set => throw new PlatformNotSupportedException();
+            get => _allowAutoRedirect;
+            set
+            {
+                CheckDisposedOrStarted();
+                _allowAutoRedirect = value;
+            }
         }
 
         public int MaxAutomaticRedirections
         {
-            get => throw new PlatformNotSupportedException();
-            set => throw new PlatformNotSupportedException();
+            get => _maxAutomaticRedirections;
+            set
+            {
+                if (value < 1)
```

Comment:
NIT:
```suggestion
                if (value <= 0)
```
Since you're throwing: `net_http_value_must_be_greater_than, 0` it makes more sense to use the same value in the condition.

Reply:
@ManickaP should this be changed as well?

https://github.com/dotnet/runtime/blob/master/src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/SocketsHttpHandler.cs#L134